### PR TITLE
Update a few things in the appdata, and add release notes for 1.3.0

### PR DESCRIPTION
**Pull request by rdbende**
@grahamvh, AFAICT you're a native English speaker. Could you review the newly added release notes for 1.3.0?

### CODE REVIEW

File: data/com.github.geigi.cozy.appdata.xml [hunk 1]
```diff
@@ -44,27 +50,26 @@
...
-  <url type="donation">https://www.patreon.com/geigi</url>
-  <url type="vcs-browser">https://github.com/geigi/cozy/</url>
-  <url type="translate">https://www.transifex.com/geigi/cozy/</url>
-  <update_contact>cozy@geigi.de</update_contact>
   <releases>
-    <release version="1.3.0" timestamp="1701613884">
+    <release version="1.3.0" date="2023-12-03">
       <description translatable="no">
+        <p>
+          After almost two years, a new version of Cozy is finally here! This release brings an updated user interface along with numerous bug fixes and improvemed performance.
```
Comment (kbdharun) [inline data/com.github.geigi.cozy.appdata.xml:57]:
```suggestion
          After almost two years, a new version of Cozy is finally here! This release brings an updated user interface along with numerous bug fixes and improved performance.
```

File: data/com.github.geigi.cozy.appdata.xml [hunk 2]
```diff
@@ -44,27 +50,26 @@
...
+        </p>
+        <p>
+          The user interface has been ported to GTK4 and Libadwaita. Thus, Cozy benefits from the new style sheet, automatic dark mode, and utilizes the latest and greatest UI elements throughout the application. Many thanks to Benedek Dévényi and grahamvh!
+        </p>
+        <p>Other changes include:</p>
         <ul>
-          <li>Update to GTK 4 and libadwaita (thank you rdbende and grahamvh!)</li>
+          <li>Much improved mobile support</li>
```
Comment (kbdharun) [inline data/com.github.geigi.cozy.appdata.xml:64]:
```suggestion
          <li>Improved mobile support</li>
```

**Review by kbdharun — COMMENTED** [review]
Hi, I noticed this PR now (sorry not the person you pinged). I have some minor suggestions for the wording. Good luck with the release.

**Comment by rdbende** [comment]
Thanks for the review @kbdharun!

**Comment by grahamvh** [comment]
Looks good to me!

File: data/com.github.geigi.cozy.appdata.xml [hunk 3]
```diff
@@ -224,11 +229,22 @@
...
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </supports>
+  <recommends>
+    <display_length compare="gt">360</display_length>
+  </recommends>
+  <branding>
+    <color type="primary" scheme_preference="light">#ffa348</color>
+    <color type="primary" scheme_preference="dark">#ffa348</color>
```
Comment (rdbende) [inline data/com.github.geigi.cozy.appdata.xml:245]: I'm not sure how this would actually look like, but these are the colors used on Cozy's icon.

Comment (geigi) [reply data/com.github.geigi.cozy.appdata.xml:245]: Let's have a look how it will look on Flathub & co :)

**Comment by geigi** [comment]
Thanks everybody, looks good to me :)

**Review by geigi — APPROVED** [review]

**Pull request merged**